scripts/build/termux_setup_self_debug.sh

```diff
@@ -0,0 +1,15 @@
+termux_print_stacktrace() {
+	local func src lineno
+	echo "💥 Error occurred at line $1 in script ${BASH_SOURCE[1]}"
+	echo "🔍 Stack trace:"
```

Comment:
This is very noisy for a debug log.
We might wanna consider text labels instead.
```suggestion
	echo "[ERROR]: ${BASH_SOURCE[1]}:$1"
	echo "[TRACE]:"
```
We should also keep the format consistent for `<source file>:<line>`.

Reply:
It is intended to be printed only once in the end of log. And I am checking solutions to merge stacktraces of all subshells into one big log so you will see only two emojis in the whole log (caused by this PR).

Reply:
Still don't like 'em.
Emoji's are harder to search for than a predefined text tag and may be broken depending on a users font setup and terminal configuration.